includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1554,11 +1555,35 @@ public function update_subscription_item( $wcpay_subscription_item_id, $data ) {
 	 * @throws API_Exception If payment method does not exist.
 	 */
 	public function get_payment_method( $payment_method_id ) {
```

Comment:
Can we maybe add some tests for this API requests?

Reply:
@zmaglica done in https://github.com/Automattic/woocommerce-payments/pull/4663/commits/8e01ddc407ddaeb4e6fbe61d80b3dcd7434d385c